File: Front/ProEventos-App/src/app/models/evento.ts
import { Lote } from "./lote";
import { Palestrante } from "./palestrante";
import { RedeSocial } from "./rede-social";

export interface Evento {
  id: number;
  local: string;
  dataEvento?: Date;
  tema: string;
  qtdPessoa: number;
  imagemURL: string;
  statusEvento: string;
  telefone: string ;
  email: string;
  lotes: Lote[];
  redeSociais: RedeSocial[];
  palestrantesEventos: Palestrante[];
}

File: Back/src/ProEventos.Domain/Evento.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using ProEventos.Domain.Identity;

namespace ProEventos.Domain
{
    public class Evento
    {
        public int Id { get; set; }
        public string Local { get; set; }
        public DateTime? DataEvento { get; set; }
        [Required]
        [MaxLength(50)]
        public string Tema { get; set; }
        public int QtdPessoas{ get; set; }
        public string ImagemURL { get; set; } 
        public string statusEvento { get; set; }
        public string Telefone{ get; set; }
        public string Email { get; set; }
        public int UserId { get; set; }
        public User User { get; set; }
        public IEnumerable<Lote> Lotes { get; set; }
        public IEnumerable<RedeSocial> RedeSociais { get; set; }
        public IEnumerable<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}